Added TimeSlice method to return total payload data size

bieryAtFnal opened the pull request:
Added a method to the TimeSlice class to return the total size of the data payloads in the fragments (useful for determing the number of TPs contained in TimeSlices in TP Stream data files).

Commit:
Added a method to the TimeSlice class to return the total size of the data payloads in the fragments (useful for determing the number of TPs contained in TimeSlices in TP Stream data files).

wesketchum reviewed (changes requested):
I like the idea. A few quick thoughts/requests:

1. If it makes sense for the TimeSlice, then I think it also makes sense for the TriggerRecord, so I would propose we should add it there as well.
2. We should add python bindings (clearly will be useful there).

Also, the name. I think it makes sense, and I can't think of something better. But at first glance, it wasn't 100% clear to me that this was _only_ the data size in the fragments, and not just the size of the fragments (ignoring the timeslice header). Again though, I think that is trying to be conveyed, and I can't think of something better, so I'm ok with it as is. Just mentioning in case anyone felt the same and has a better idea.

Commit:
Changed the name of the new TimeSlice method to get_sum_of_fragment_payload_sizes, also added it to the TriggerRecord class, added tests of the new method and get_total_size_bytes to the TimeSlice and Trigger record unit tests, and added the two 'size' methods to the python bindings for TimeSlice and TriggerRecord.

bieryAtFnal commented:
I changed the name of the new method to get_sum_of_fragment_payload_sizes.  Hopefully that makes its purpose more clear.

I also propagated the new method to the TriggerRecord class.  
Also, I added unit tests for the existing get_total_data_size() method and the new method.
And, I added both to the Python bindings.
